tests/test_androidmodule_ctypes_finder.py

```diff
@@ -0,0 +1,137 @@
+
+from mock import MagicMock
+import os
+import shutil
+import sys
+import tempfile
+
+sys.modules["jnius"] = MagicMock()  # fake jnius used later
+
+
+# Import the tested android._ctypes_library_finder module,
+# making sure android._android won't crash us!
+# (since android._android is android-only / not compilable on desktop)
+android_module_folder = os.path.abspath(os.path.join(
+    os.path.dirname(__file__),
+    "..", "pythonforandroid", "recipes", "android", "src"
+))
+sys.path.insert(0, android_module_folder)
```

Comment:
would it really hurt to `pythonforandroid/recipes/android/src/__init__.py` so we can skip that thing above for Python2?

Reply:
@AndreMiras dunno it just seems uglier to me, messing around in the filesystem etc. (and not in a temp folder) but that's just my personal impression of course

Reply:
hm OK let's leave it like this for now then, will cry in silence over there :door:  :cry:

Reply:
@AndreMiras aww nooo don't be sad :cry: oh noo

Reply:
ahah I'm fine actually I agree the `__init__.py` is not super satisfying either so one hack or the other I'm fine